### PR TITLE
Add Asphalt 3D

### PR DESCRIPTION
**Pull request by danzel**
Adds Asphalt 3D.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/asphalt-3d/game.dat [hunk 1]
```diff
@@ -0,0 +1,29 @@
+title = "Asphalt 3D"
+description = "Asphalt 3D is a racing game, in which the player must maneuver an automobile to compete against computer-controlled drivers on various race tracks. Players earn rewards such as car upgrades and XP points for winning races throughout the game, and reaching higher levels allows them to use better sports cars to unlock."
```
Comment (BreadFish64) [inline games/asphalt-3d/game.dat]: include the developer/publisher

**Comment by danzel** [comment]
Description updated and more screenshots added.

**Comment by danzel** [comment]
Not totally sure what was wrong with the wiki, it matched the format afaict.
Did a pass on it anyway, please review :bow:

**Comment by Schplee** [comment]
@danzel 

Current Wiki: 
`Asphalt 3D is quite slow to load up. Once in game it runs ok, lagging when you are near a lot of other cars. Graphics and audio seem correct.`

Following Contribution Guidelines format:
`Asphalt 3D has some problems in Citra. Graphically, the game suffers from minor issues, but requires powerful hardware to obtain decent speeds. It may suffer from minor audio issues at time, but this does not hinder gameplay in any way. You may experience slow down when booting the game and when there is a lot of on screen activity.`

**Comment by danzel** [comment]
Changed

**Comment by Schplee** [comment]
LGTM

**Pull request merged**